spring-boot-starter-example/README.md

```diff
@@ -30,13 +30,6 @@ mvn -v
 	```
 
 4. Run spring-boot-starter-example (from the spring-boot-starter-example dir)
-
-	```sh
-	java -jar target/moesif-spring-boot-starter-example.jar
```

Comment:
The jar file is not generated with the build with current settings/instructions.
bot the below instructions work fine.